mozilla/persona
# Re-enable unchecking the "Always sign in using this email..." checkbox.

Using @fawek's pull request as a base for a full solution.
- If the user clicks on the label, proxy the click event to the input element.
- If the user clicks on the input element, let the browser handle things.
- Add user tests to check clicks on both labels and input elements.

Things to test:
- Ensure capability of selection radio/checkboxes themselves.
- Ensure capability of selecting radio/checkboxes when clicking on their labels - extra attention to iOS.
- Ensure toggle capability when clicking on label for "Always sign in using this email..." label.

issue #1155
